SystemReference/CMakeLists.txt

```diff
@@ -39,6 +39,9 @@ include("${FPRIME_FRAMEWORK_PATH}/cmake/FPrime-Code.cmake")
 # core components in the topology, which is also added here.
 ##
 
+# needed for libcamera
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Can this be set just on the libcamera or is this needed everywhere?

Reply:
I don't think we need it everywhere, what do you mean by "set just on the libcamera"?